Source/WebInspectorUI/Localizations/en.lproj/localizedStrings.js

```diff
@@ -194,6 +194,8 @@ localizedStrings["Anonymous Script %d"] = "Anonymous Script %d";
 localizedStrings["Anonymous Scripts"] = "Anonymous Scripts";
 localizedStrings["Anonymous Style Sheet %d"] = "Anonymous Style Sheet %d";
 localizedStrings["Anonymous Style Sheets"] = "Anonymous Style Sheets";
+/* Header for section with appearance user preferences. */
+localizedStrings["Appearance @ User Preferences Overrides"] = "Appearance";
```

Comment:
It feels odd that one section would be labeled "Appearance", and the other section "Accessibility Preferences". I'd propose changing the name of the Ax section to just be "Accessibility" for balance, since putting "Preferences" after each section title seems redundant.